adapters.json

```diff
@@ -34,5 +34,10 @@
       "appnexus": {
         "alias": "pagescience"
       }
+    },
+    {
+      "appnexusAst": {
+        "supportedMediaTypes": ["video"]
+      }
```

Comment:
@protonate 
What are feelings on this pattern? Supported types specified at the configuration level...